src/Exports.jl

```diff
@@ -745,6 +745,7 @@ export is_non_zero_divisor
 export is_normal
 export is_normal_subgroup
 export is_normalized_by
+export is_one
```

Comment:
These are already exported in `AbstractAlgebra` so these exports are not strictly needed.
What is our policy here @fingolfin ?